en-US/Translation.xml

```diff
@@ -38,7 +40,7 @@
             <term>Ordered lists</term>
             <listitem>
                 <para>
-                    They appear as a numbered list.
+                    These appear as numbered lists.
```

Comment:
For the sake of translation, "These" should be followed by a noun wherever it occurs.

Reply:
Well, this guide is not translated, but ok.

Reply:
tbh I disagree with that section in the current guide. I don't remember reviewing or ack'ing it (doesn't mean I didn't) but a) it's not a very good title, and b) I don't think following nouns are always required. I think this small section is far to prescriptive, leaving no room for other possibilities, and being a lot more strict than "providing guidelines", which is what the style guide is for in many cases.